infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -557,6 +558,7 @@ EOT
       secured_variables : [
         { name : "ACCESS_TOKEN", writable : true },
         { name : "REFRESH_TOKEN", writable : true },
+        { name : "OAUTH_REFRESH_TOKEN", writable : true, lockable : true },
```

Comment:
@aperez-worklytics note this is the solution Jira/Zoom locking; please take a good look, and we should try to test

Reply:
Yeah that the thing I was to asking you; ok for that solution but we'll need to update this on java side too